.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,47 @@
+**Is it a ?**
+
+- [ ] Bug Report
```

Comment:
Please do not nest. Just keep the checkboxes above and then provide the templates with `###` heading below in order

- [ ] X
- [ ] Y
- [ ] Z

### X

Template

### Y

Template

### Z

Template

Reply:
What about something that is common for eg. "Would you like to work on it ?"

Reply:
Yeah, that should be added

Reply:
I meant where. Should i add them in each of the three sections ?

Reply:
No, just at the bottom